src/components/pages/Theme/Theme.jsx

```diff
@@ -412,13 +422,53 @@ export const Theme = () => {
           </Header>
           <MessageExample
             color='red'
-            content={messageText}
+            content={contentText}
             icon='warning circle'
             size='mini'
           />
         </Segment>
       </div>
 
+      <div className='section'>
+        <Segment padded>
+          <Header as='h2' dividing>
+            Modal
+          </Header>
+
+          <Button onClick={handleOpenModal}>Open Modal</Button>
+
+          <Header size='small' dividing>
+            Modal trigger
+          </Header>
+          <Modal
+            onClose={handleCloseModal}
+            onOpen={handleOpenModal}
+            open={isModalOpen}
+            trigger={<Button>Open Modal (trigger)</Button>}
+            closeIcon
+          >
+            <Modal.Header>
+              <Header as='h2'>Heading 2</Header>
+            </Modal.Header>
+
+            <Modal.Content>
+              <Modal.Description>
```

Comment:
We just need to remove the `margin` on the `.description`. It seems to only be applied at smaller screen sizes but we can remove it entirely.
![Screenshot 2023-06-14 at 12 34 49 PM](https://github.com/abby-moffitt/GA-Semantic/assets/1568524/9c2a2a52-d6a5-412a-986e-2481bb88aa59)

Reply:
Everything on smaller screens has `!important`, so it's hard to change. I'll look for a way to override this.

Reply:
@michael-lynch please take a look. At first, I thought of adjusting the value by adding _margin_ around it ti match `--spacing-3`. That didn't work because on smaller screens they have `!important` for both _padding_ and _margin_ 😔
So, the only way to go about it was to override using `!important`. I just don't like the idea of overriding _important_ with _important_...